assembl/static2/css/components/announcement.scss

```diff
@@ -0,0 +1,83 @@
+.announcement {
+  text-align: center;
+
+  .announcement-title {
+    .announcement-title-text {
+      margin-top: 0.5em;
```

Comment:
Why do you use `em`?

Reply:
Because I thought %/vw/vh > em > px

Reply:
Is this bs? https://gist.github.com/basham/2175a16ab7c60ce8e001